examples/neq-switching/run_nonequilibrium.py

```diff
@@ -9,13 +9,13 @@
 from perses.dispersed.feptasks import compute_reduced_potential
 import numpy as np
 
-def traj_frame_to_sampler_state(traj: md.Trajectory, frame_number: int):
+def traj_frame_to_sampler_state(traj: md.Trajectory, frame_number: int,box_vectors):
     xyz = traj.xyz[frame_number, :, :]
     box_vectors = traj.openmm_boxes(frame_number)
-    sampler_state = states.SamplerState(unit.Quantity(xyz, unit=unit.nanometers), box_vectors=box_vectors)
+    sampler_state = states.SamplerState(unit.Quantity(xyz, unit=unit.nanometers))
```

Comment:
This looks like a mistake---why would we accept `box_vectors` as an argument, retrieve it from `traj`, and then not include it in the `sampler_state`?

Reply:
TODO:
* Add a docstring
* remove `box_vectors` from function arguments
* keep `box_vectors` argument for `states.SamplerState`